modules/local/read_counts.nf

```diff
@@ -0,0 +1,30 @@
+process READ_COUNTS {
+    label 'process_low'
+
+    conda "conda-forge::perl=5.26.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/perl:5.26.2' :
+        'biocontainers/perl:5.26.2' }"
+
+    input:
+    path raw_fastqc
+    path trim_fastqc
+    path preextract_fastqc
+    path correct_tsv
+
+    output:
+    path "read_counts.csv"        , emit: read_counts
```

Comment:
please also output the perl version into version.yml for the sake of reproducibility

Reply:
Resolved with commit 22c10cc